.docker/startup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This file is part of the Kimai time-tracking app.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+
+until mysql -u lamp -plamp -h db lamp -e "show tables"; do
+  >&2 echo "Mysql is unavailable - sleeping"
+  sleep 5
+done
+
+APP_PATH=/var/www/html
+
+# Check the mount point for the code base, this could be pretty much any file
+if [ ! -e $APP_PATH/.env ]; then
+    # Copy in the code base so it's on a shared volume for php-fpm and nginx
+    rm -rf /var/www/html/{*,.*}
+    cp -r /var/tmp/kimai/* $APP_PATH
+    cp -r /var/tmp/kimai/.??* $APP_PATH
+fi
+
+# If the schema does not exist then create it (and run the migrations)
+TABLE_COUNT=$(/var/www/html/bin/console doctrine:query:sql "select * from kimai2_users")
```

Comment:
what if someone changes the database table prefix from kimai2_ to something else?
Probably this is more fail-safe: `bin/console doctrine:query:dql "Select u from App\Entity\User u"`

Reply:
A better solution.